aionanoleaf/typing.py

```diff
@@ -85,6 +85,7 @@ class InfoData(TypedDict):
     serialNo: str
     manufacturer: str
     firmwareVersion: str
+    hardwareVersion: str
```

Comment:
This is not used anywhere, which is why I did not update it

Reply:
It's used here:
https://github.com/milanmeu/aionanoleaf/blob/c1be410702839f39b702349546da6f1bdd16d9ff/aionanoleaf/nanoleaf.py#L247

Reply:
Nope

Reply:
This is required for mypy.

Reply:
:confused: :question: 
`mypy aionanoleaf` output:
```python3
aionanoleaf/nanoleaf.py:257: error: TypedDict "InfoData" has no key "hardwareVersion"
```

Reply:
No place indicated that you had that as a requirement.
I'll try again later once the repo is ready for contributions.

Reply:
I'm sorry this wasn't clear. Meanwhile I added CI with GitHub Actions for mypy and flake8. I will edit the README for this as well.